deploy/LOCAL_INSTALL.md

```diff
@@ -37,7 +37,34 @@ docker-compose exec seqr python manage.py createsuperuser  # create a seqr Admin
 open http://localhost     # open the seqr landing page in your browser. Log in to seqr using the email and password from the previous step
 ```
    
-#### Annotating and loading VCF callsets - option #1: annotate on a Google Dataproc cluster, then load in to an on-prem seqr instance 
+#### Annotating and loading VCF callsets 
```

Comment:
How about moving this title just before `Option #1`? And, add a title saying `Prepare for annotation` here?

Reply:
given the rest of the file, this large header needs to stay as-is or it will be really confusing. I will add a sub-header here like you propose

Reply:
updated